test/unit/scan/test_scan_policies.py

```diff
@@ -7,14 +7,207 @@
 # License https://www.backblaze.com/using_b2_code.html
 #
 ######################################################################
+from pathlib import Path
+import platform
 import re
-
+from unittest.mock import MagicMock
+from b2sdk.scan.folder import LocalFolder
+from b2sdk.utils import fix_windows_path_limit
 import pytest
 
 from apiver_deps import ScanPoliciesManager
 from apiver_deps_exception import InvalidArgument
 
 
+class TestFolderTraversal:
```

Comment:
does this belong in `test_scan_policies.py`? I think we might need a new file like `test_folder_traversal.py`or something